<launch>

    <!-- Node for stereo image splitter -->
    <node name="stereo_image_splitter" pkg="image_splitter_ros" type="stereo_image_splitter_node" output="screen">
        <param name="is_grey" value="True" />
        <remap from="stitched_images" to="/libcamera_ros/image_raw" />
        <remap from="left_image" to="/left_image" />
        <remap from="right_image" to="/right_image" />
    </node>

</launch>
